Dockerfile

```diff
@@ -4,6 +4,10 @@ ARG ANSIBLE_VERSION=2.9.10
 ARG MOLECULE_VERSION=3.0.7
 ARG YUM_REPOSITORY=yum-repository.platform.aws.chdev.org
 
+RUN cd /etc/yum.repos.d/
+RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-*
+RUN sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```

Comment:
The `cd` command isn't required here as the following `sed` commands operate within that directory anyway. So this can be compressed to:

```
RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* && \
    sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-*
```